src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-strict.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    <p>Select to fail the test suite on warnings</p>  
```

Comment:
This looks like the only one without a period at the end.

Reply:
Done.